d8s_math/maths.py

```diff
@@ -688,3 +688,15 @@ def number_to_engineering_notation(number):
 
     decimal_form = decimal.Decimal(number)
     return decimal_form.normalize().to_eng_string()
+
+def hex_get_bytes(hex_number,number_of_bytes):
```

Comment:
Minor style improvement:

```suggestion
def hex_get_bytes(hex_number, number_of_bytes):
```